.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: updatecli
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Run every Monday at 9 AM
+    - cron: '0 9 * * 1'
+  repository_dispatch:
+    types: [epinio-release]
+
+permissions:
+  contents: write
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Apply
+        uses: updatecli/updatecli-action@v1.17.0
+        if: github.ref == 'refs/heads/main'
```

Comment:
I'd move that to `jobs.updatecli.if`, so the checkout step is not run

Reply:
Thanks for the very good suggestion